core/src/core-api/config.js

```diff
@@ -279,6 +280,32 @@ class LuigiConfig {
     this.unload();
     this.setConfig(cfg);
   }
+
+  /**
+   * Delete cache. Clear titleResolver cache of nested navigation nodes.
+   * @memberof Configuration
+   * @example
+   * Luigi.clearNavigationCache();
+   * @since NEXT_RELEASE
+   */
+  clearNavigationCache() {
+    NodeDataManagementStorage.deleteCache();
+
+    const clearTitleResolverCache = nodes => {
+      if (nodes && nodes.forEach) {
+        nodes.forEach(node => {
+          if (node.titleResolver && node.titleResolver._cache) {
+            node.titleResolver._cache = undefined;
+          }
+          if (node.children) {
+            clearTitleResolverCache(node.children);
+          }
+        });
+      }
+    };
+
+    clearTitleResolverCache(this.getConfig().navigation.nodes);
+  }
 }
```

Comment:
(q) We clear the cache of the nested navigation nodes - shouldn't we return a warning of some kind of any of the nodes do not meet the criteria (i.e. titleResolver doesn't exist in the node) within the ```if```s? :)